### PR TITLE
Update GCP cloud libraries BOM to 26.8.0

### PR DESCRIPTION
**Pull request by tonytanger**
Update GCP cloud libraries BOM to 26.8.0

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by tonytanger** [comment]
@pabloem @suztomo

File: buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy [hunk 1]
```diff
@@ -608,10 +608,10 @@ class BeamModulePlugin implements Plugin<Project> {
         google_cloud_firestore                      : "com.google.cloud:google-cloud-firestore", // google_cloud_platform_libraries_bom sets version
         google_cloud_pubsub                         : "com.google.cloud:google-cloud-pubsub", // google_cloud_platform_libraries_bom sets version
         google_cloud_pubsublite                     : "com.google.cloud:google-cloud-pubsublite",  // google_cloud_platform_libraries_bom sets version
-        // The GCP Libraries BOM dashboard shows the versions set by the BOM:
-        // https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/26.5.0/artifact_details.html
+        // The release notes shows the versions set by the BOM:
+        // https://github.com/googleapis/java-cloud-bom/releases/tag/v26.8.0
```
Comment (suztomo) [inline buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:612]: Thank you. This GitHub release page is updated quicker than previous one and holds the library versions in the BOM.

**Review by suztomo — APPROVED** [review]

File: buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy (hunk 1 above)
Comment (suztomo) [reply buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:612]: > This release is the same as previous release (https://github.com/googleapis/java-cloud-bom/releases/v26.7.0) except google-cloud-bigtable version update to 2.19.0 (prev: 2.18.4).

This matches what Mattie wants.

**Comment by tonytanger** [comment]
Run Java_PVR_Flink_Batch PreCommit

**Comment by tonytanger** [comment]
Run Java_PVR_Flink_Batch PreCommit

**Comment by pabloem** [comment]
LGTM. I'll merge this as is since it's a critical change and an extensive set of test suites has passed on jenkins

**Pull request merged**